src/index.js

```diff
@@ -1,12 +1,33 @@
-import React from 'react'
-import ReactDOM from 'react-dom'
+import React from "react";
+import ReactDOM from "react-dom";
+import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
 
-import Navbar from './components/Navbar'
+import Navbar from "./components/Navbar";
+import Home from "./views/Home";
+import About from "./views/About";
+import Events from "./views/Events";
+import PassionInAction from "./views/PassionInAction";
+import Join from "./views/Join";
+import Hosts from "./views/Hosts";
 
 function Site() {
   return (
-    <Navbar />
-  )
+    <Router>
```

Comment:
To make react router work on subdirectories (for dev deployment), we'll have to set `basename` argument to the full subdir path, but I'll take care of that in a following PR.

Reply:
Done